src/app/views/account/account.controller.coffee

```diff
@@ -90,6 +90,20 @@ angular.module 'mnoEnterpriseAngular'
       vm.isLocalizationOpen = false
 
       # ----------------------------------------------------
+      # Developer Section
+      # ----------------------------------------------------
+      vm.isDevOpen = false
+      vm.user.registerDeveloper = ->
+        MnoeCurrentUser.registerDeveloper(vm.user.model).then(
```

Comment:
Why do you pass an object? `MnoeCurrentUser.registerDeveloper()` doesn't have a param in its signature.